README.rst

```diff
@@ -0,0 +1,44 @@
+=========
+ REANA DB
+=========
+
+.. image:: https://img.shields.io/travis/reanahub/reana-db.svg
+      :target: https://travis-ci.org/reanahub/reana-db
+
+.. image:: https://img.shields.io/coveralls/reanahub/reana-db.svg
+      :target: https://coveralls.io/r/reanahub/reana-db
+
+.. image:: https://readthedocs.org/projects/docs/badge/?version=latest
+      :target: https://reana-db.readthedocs.io/en/latest/?badge=latest
+
+.. image:: https://badge.waffle.io/reanahub/reana.svg?label=Status%3A%20ready%20for%20work&title=Issues%20ready%20for%20work
+      :target: https://waffle.io/reanahub/reana
+
+.. image:: https://badges.gitter.im/Join%20Chat.svg
+      :target: https://gitter.im/reanahub/reana?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge
+
+.. image:: https://img.shields.io/github/license/reanahub/reana-db.svg
+      :target: https://github.com/reanahub/reana-db/blob/master/COPYING
+
+About
+-----
+
+REANA-DB is a component of the `REANA
+<http://reanahub.io/>`_ reusable analysis platform.
+It contains REANA database models and utilities.
+
+Features
+--------
+
+- FIXME
```

Comment:
- database persistence for REANA system
- database models and utilities
- database upgrades and migrations

perhaps?